ime/app/src/main/java/com/anysoftkeyboard/keyboards/views/AnyKeyboardViewBase.java

```diff
@@ -2164,6 +2176,20 @@ private void updatePrefSettings(final String overrideValue) {
         }
     }
 
+    private void updatePrefSettings_hint_size(final String overrideValue) {
```

Comment:
Can you use the standard Java naming here? `updatePrefSettingsHintTextSizeFactor`

Reply:
Will change it